src/main/java/org/mskcc/cbio/oncokb/web/rest/CronJobController.java

```diff
@@ -84,9 +79,10 @@
 
     private final ApplicationProperties applicationProperties;
 
-    final String USERS_USAGE_SUMMARY_FILE = "usage-analysis/userSummary.json";
-    final String RESOURCES_USAGE_SUMMARY_FILE = "usage-analysis/resourceSummary.json";
-    final String RESOURCES_USAGE_DETAIL_FILE = "usage-analysis/resourceDetail.json";
+    final String USERS_USAGE_SUMMARY_FILE = "public-website/usage-analysis/userSummary.json";
+    final String RESOURCES_USAGE_SUMMARY_FILE = "public-website/usage-analysis/resourceSummary.json";
+    final String RESOURCES_USAGE_DETAIL_FILE = "public-website/usage-analysis/resourceDetail.json";
```

Comment:
Could you move these to constants.java and update the reference in the `UsageAnalysisController`?